controllers/articles.js

```diff
@@ -1,9 +1,24 @@
 
 var mongoose  = require('mongoose'),
-  moment    = require('moment'),
-  Article   = mongoose.model('Article');
+  moment      = require('moment'),
+  Article     = mongoose.model('Article'),
+  passport    = require('passport'),
+  express     = require('express'),
+  auth        = require('./../middlewares/auth');
 
-exports.view = function (req, res) {
+module.exports = (function () {
```

Comment:
Because I don't like code in global scope (expect for imports). I agree this one is a little far fetched.